lib/createForm.js

```diff
@@ -4,6 +4,30 @@ import { v4 as uuid } from 'uuid';
 import { APPLICATION_GRAPH, FORM_STATUS_CONCEPT, PREFIXES } from '../config';
 import { bindingsToNT } from '../utils/bindingsToNT';
 
+export async function createEmptyForm() {
+  const publicServiceId=uuid();
+  const publicServiceUri=`<http://data.lblod.info/id/public-service/${publicServiceId}>`
+
+  const now=new Date().toISOString();
+  const query=`
+    INSERT DATA {
```

Comment:
Is it possible to use the prefixes?

Reply:
tried it but it caused some errors (see mu-semtech chat)...

Reply:
the problem was that \t s were in the prefixes.... why did everything else work then.... im big confused... well whatever it works now